.gitlab/shared.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+.get_artifactory_token_linux: &get_artifactory_token_linux
+  - ARTIFACTORY_TOKEN=$(aws ssm get-parameter --region us-east-1 --name $ARTIFACTORY_TOKEN_SSM_NAME --with-decryption --query "Parameter.Value" --out text)
+
+.get_artifactory_token_win: &get_artifactory_token_win
+  - $ARTIFACTORY_TOKEN=$(aws ssm get-parameter --region us-east-1 --name $ARTIFACTORY_TOKEN_SSM_NAME --with-decryption --query "Parameter.Value" --out text)
+
+.setup_ruby_mirror_linux:
+  - set +x
+  - *get_artifactory_token_linux
+  - if [ "${USE_CACHING_PROXY_RUBY}" = "true" ]; then export BUNDLE_MIRROR__RUBYGEMS__ORG=https://${ARTIFACTORY_USERNAME}:${ARTIFACTORY_TOKEN}@${ARTIFACTORY_URL}/${ARTIFACTORY_GEMS_PATH}; fi
```

Comment:
That's an environment variable recognized by Bundler and they decided to require usage of two underscores. I honestly don't know why, but it won't work otherwise.